### DISCUSSION TITLE
Table and views open very slow, any idea?

### DISCUSSION THREAD

**Discussion by sicilian-lizard** [pawelsalawa/sqlitestudio#3895] [Q&A]
I can't tell since when but now it takes around 10 seconds  or more to open a table or a view, regardless of size, while tables open fine when the program starts.

Any idea?

When can we expect a new version?

**Comment by pawelsalawa** [comment]
It's hard to say what is the reason for slow opening. Does this happen repeatedly? I mean when you restart the app, does this happen again after some time? How long after it appears? Minutes? Hours? Days? Weeks?

Regarding new version - unfortunately I cannot give an estimate... few months back I though I will have more free time now, but actually I still don't :/
